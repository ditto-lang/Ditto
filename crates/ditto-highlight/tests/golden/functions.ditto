module Test exports (
    five,
);


five = 5;
