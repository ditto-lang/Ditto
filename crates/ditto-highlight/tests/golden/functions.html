<KeywordImport>module</KeywordImport> <Namespace>Test</Namespace> <KeywordImport>exports</KeywordImport> <Bracket>(</Bracket>
    <TopLevelName>five</TopLevelName><Delimiter>,</Delimiter>
<Bracket>)</Bracket><Delimiter>;</Delimiter>


<TopLevelName>five</TopLevelName> = <Int>5</Int><Delimiter>;</Delimiter>
